test/unit/api/connectionTest.js

```diff
@@ -20,6 +20,62 @@ describe('api key', () => {
   })
 })
 
+describe('hash', () => {
+  const query1 = `
+  {
```

Comment:
It is very hard to spot the difference between the two queries. 
Extract to function: 
```js
function buildQuery (to) {
return `
  {
    projectBySlug(slug: "santiment"){
    ethTopTransactions(from: "2020-02-02T00:00:00Z",
      to: ${to},
      limit: 155,
      transactionType: ALL) {
        datetime
        fromAddress{
          address
          isExchange
        }
        toAddress{
          address
          isExchange
        }
        trxHash
        trxValue
      }
    }
  }`
}
```
and then:

```js
const query1 = buildQuery("2020-02-15T00:00:00Z")
const query2 = buildQuery("2020-02-15T05:00:00Z")
```